sdk/src/client/secret/ledger_nano.rs

```diff
@@ -159,7 +160,7 @@ impl SecretManage for LedgerSecretManager {
         let lock = self.mutex.lock().await;
 
         // get ledger
-        let ledger = get_ledger(coin_type, bip32_account, self.is_simulator).map_err(Error::from)?;
+        let ledger = get_ledger(coin_type, Protocols::Nova, bip32_account, self.is_simulator).map_err(Error::from)?;
```

Comment:
Nothing that should be blocking now but if ever you want to change in a future version: enum names are usually singular in Rust

Reply:
And I would personally put that parameter first

Reply:
Ok, changed :blush:

Reply:
🙏🏻